src/coreclr/pal/tests/palsuite/compilableTests.txt

```diff
@@ -772,13 +772,11 @@ threading/ResetEvent/test2/paltest_resetevent_test2
 threading/ResetEvent/test3/paltest_resetevent_test3
 threading/ResetEvent/test4/paltest_resetevent_test4
 threading/ResumeThread/test1/paltest_resumethread_test1
-threading/SwitchToThread/test1/paltest_switchtothread_test1
```

Comment:
Is this related?

Reply:
This line is redundant - the same line is ~10 lines below already. I think this was just an incorrectly resolved conflict in the original PR.